fusion/README.md

```diff
@@ -0,0 +1,60 @@
+# Ansible Playbooks for the Fusion DevKit
+
+## Summary
+The goal of this document is to help you set up and understand the functionality and scope of the ansible playbooks here providers.
```

Comment:
providers -> provided?

Reply:
I haven't checked README as I only moved it from on repo to another.
But I can go through and check any typos then

Reply:
Went through README and fixed all typos I could find